Source/WTF/wtf/text/WTFString.h

```diff
@@ -573,7 +573,6 @@ inline String operator"" _str(const UChar* characters, size_t length)
 
 } // namespace WTF
 
-using WTF::KeepTrailingZeros;
```

Comment:
WTF design has been to use these `using` statements rather than `WTF::` prefixes at site of use. I know this is unconventional, but it’s the pattern we use in most cases across the library. To do that we would have `using WTF::TrailingZerosPolicy` here and fewer (maybe no) `WTF::` prefixes elsewhere.

I’d be open to changing the WTF approach, and if we’d agreed that we’d not do it for anything new that would be OK with me.

But if I was making this change I would have stayed consistent with the approach and done the above.

Reply:
Fair! Created #17375.